cmd/contour/servecontext.go

```diff
@@ -349,15 +353,18 @@ func (ctx *serveContext) tlsconfig() *tls.Config {
 	}
 
 	// Attempt to load certificates and key to catch configuration errors early.
-	_, err = loadConfig()
-	check(err)
+	if _, lerr := loadConfig(); lerr != nil {
+		log.WithField("context", "loadConfig").Fatalf("err: %v", err)
+	}
 
 	return &tls.Config{
 		ClientAuth: tls.RequireAndVerifyClientCert,
 		Rand:       rand.Reader,
 		GetConfigForClient: func(*tls.ClientHelloInfo) (*tls.Config, error) {
 			config, err := loadConfig()
-			check(err)
+			if err != nil {
+				log.Fatalf("err: %v", err)
+			}
```

Comment:
I think that we we can probably just drop this check and pass the error up to the caller.

Reply:
Done